c_src/membrane_rtmp_plugin/sink/rtmp_sink.c

```diff
@@ -7,10 +7,15 @@ void handle_init_state(State *state);
 
 void handle_destroy_state(UnifexEnv *env, State *state);
```

Comment:
I know that it has been there before, but we probably don't need the function forward declaration as it is already present in auto-generated code, am I right?